examples/conf.py

```diff
@@ -69,6 +69,7 @@
         {"name": "API", "url": "https://docs.pymc.io/en/stable/api.html"},
     ],
     "page_sidebar_items": ["postcard", "page-toc", "edit-this-page"],
+    "google_analytics_id": "G-6KPRBTE6WV",
```

Comment:
Which analytics account does this record to?

Reply:
there are 3 "properties" inside the pymc devs google account: versioned docs, pymcon and examples (this one)